Lib9c/Action/RankingBattle.cs

```diff
@@ -16,7 +16,7 @@
 namespace Nekoyume.Action
 {
     [Serializable]
-    [ActionType("ranking_battle")]
+    [ActionType("ranking_battle4")]
```

Comment:
`RankingBattle4` ActionType is `ranking_battle`. `RankingBattle` ActionType is `ranking_battle4`.
Actiontypes seem to have been reversed. Is there any reason?